lib/servo.js

```diff
@@ -275,8 +274,8 @@ Servo.prototype.to = function(degrees, time, rate) {
   if (this.invert) {
     degrees = Board.map(
       degrees,
-      this.range[0], this.range[1],
-      this.range[1], this.range[0]
+      0, 180,
+      180, 0
```

Comment:
> If the values in range are not equidistant from the center position of the servo it will not be mapped correctly.

I wonder if this is what caused me headaches with some of the bipeds I built—in most cases I was specifying an artificial range.